internal/commands/kubernetes/show.go

```diff
@@ -0,0 +1,114 @@
+package kubernetes
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/UpCloudLtd/upcloud-cli/v2/internal/commands"
+	"github.com/UpCloudLtd/upcloud-cli/v2/internal/completion"
+	"github.com/UpCloudLtd/upcloud-cli/v2/internal/format"
+	"github.com/UpCloudLtd/upcloud-cli/v2/internal/output"
+	"github.com/UpCloudLtd/upcloud-cli/v2/internal/resolver"
+	"github.com/UpCloudLtd/upcloud-cli/v2/internal/ui"
+
+	"github.com/UpCloudLtd/upcloud-go-api/v4/upcloud/request"
+)
+
+// ShowCommand creates the "kubernetes show" command
+func ShowCommand() commands.Command {
+	return &showCommand{
+		BaseCommand: commands.New(
+			"show",
+			"Show load balancer details",
+			"upctl kubernetes show 55199a44-4751-4e27-9394-7c7661910be3",
+			"upctl kubernetes show my-load-balancer",
+		),
+	}
+}
+
+type showCommand struct {
+	*commands.BaseCommand
+	resolver.CachingKubernetes
+	completion.Kubernetes
+}
+
+// Execute implements commands.MultipleArgumentCommand
+func (s *showCommand) Execute(exec commands.Executor, uuid string) (output.Output, error) {
+	svc := exec.All()
+	cluster, err := svc.GetKubernetesCluster(&request.GetKubernetesClusterRequest{UUID: uuid})
+	if err != nil {
+		return nil, err
+	}
+
+	var networkName string
+	if network, err := svc.GetNetworkDetails(&request.GetNetworkDetailsRequest{UUID: cluster.Network}); err != nil {
+		networkName = ""
+	} else {
+		networkName = network.Name
+	}
+
+	nodeGroupRows := []output.TableRow{}
+	for _, nodeGroup := range cluster.NodeGroups {
+		kubeletArgs := make([]string, 0)
+		for _, v := range nodeGroup.KubeletArgs {
+			kubeletArgs = append(kubeletArgs, fmt.Sprintf("Key: %s\nValue: %s", v.Key, v.Value))
+		}
+		labels := make([]string, 0)
+		for _, v := range nodeGroup.Labels {
+			labels = append(labels, fmt.Sprintf("Key: %s\nValue: %s", v.Key, v.Value))
+		}
+		taints := make([]string, 0)
+		for _, v := range nodeGroup.Taints {
+			taints = append(taints, fmt.Sprintf("Key: %s\nValue: %s\nEffect: %s", v.Key, v.Value, v.Effect))
+		}
+		nodeGroupRows = append(nodeGroupRows, output.TableRow{
+			nodeGroup.Name,
+			nodeGroup.Count,
+			nodeGroup.Plan,
+			nodeGroup.Storage,
```

Comment:
I think we have helper function that could be used to display storage name instead of UUID in human readable output, but that could be introduced here also in separate PR.